packages/create-bison-app/template/graphql/modules/user.ts.ejs

```diff
@@ -53,6 +53,27 @@ export const UserQueries = extendType({
       description: 'Returns the currently logged in user',
       resolve: (_root, _args, ctx) => ctx.user,
     });
+
+    // List Users Query (admin only)
+    t.field('users', {
+      type: list('User'),
+      resolve: async (_root, _args, ctx) => {
+        if (!isAdmin(ctx.user)) throw new ForbiddenError('Unauthorized');
+
+        return await ctx.db.user.findMany({ ..._args });
+      },
+    });
+
+    // User Query
+    t.field('user', {
+      type: 'User',
+      args: {
+        where: nonNull(arg({ type: 'UserWhereUniqueInput' })),
+      },
+      resolve: async (_root, _args, ctx) => {
```

Comment:
I think we prefix the parameters with an underscore only if it's unused so the linter won't complain about it. If that's true, `_args` should be changed to `args` in this case and also in the `users` query.